<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件管理</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>磁盘利用情况</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="filelist">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>241</width>
      <height>581</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>文件列表</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="freetableView">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>30</y>
      <width>191</width>
      <height>581</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <column/>
    <column/>
   </widget>
   <widget class="QTextBrowser" name="bitmapview">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>30</y>
      <width>191</width>
      <height>581</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Heiti SC</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="viewtype">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>空闲表</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>91</width>
      <height>173</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="freetable">
       <property name="text">
        <string>空闲表法</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="freelinklist">
       <property name="text">
        <string>空闲链表法</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="bitmap">
       <property name="text">
        <string>位示图法</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>260</y>
      <width>111</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="run">
       <property name="text">
        <string>创建随机文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>↓</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="del">
       <property name="text">
        <string>删除奇数文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>↓</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addMore">
       <property name="text">
        <string>添加5个文件</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>371</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="zoomScale">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>22</width>
      <height>581</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>970</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
